src/stories/components/Toast.stories.tsx

```
@@ -0,0 +1,81 @@
+  tags: ["autodocs"],
+} satisfies Meta<typeof ToastWithTrigger>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+// 에러 토스트 스토리
+export const Error: Story = {
+  args: {
+    title: "에러 발생!",
+    description: "문제가 발생했습니다. 다시 시도해주세요.",
+    type: "error" ,
```

Biome (2.3.14)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)